brick/brick.go

```diff
@@ -0,0 +1,88 @@
+package brick
+
+import (
+	"crypto/md5"
+	"fmt"
+	"path"
+	"strings"
+
+	"github.com/gluster/glusterd2/utils"
+	"github.com/gluster/glusterd2/volume"
+)
+
+const (
+	// TODO: Remove hardcoding
+	//	glusterfsd = "/usr/local/sbin/glusterfsd"
+	glusterfsd = "/usr/bin/sleep"
+)
+
+type Brick struct {
+	// Externally consumable using methods of Daemon interface
+	binarypath     string
+	args           string
+	socketfilepath string
+	pidfilepath    string
+
+	// For internal use
+	brickinfo volume.Brickinfo
```

Comment:
You could also just implement these interface functions directly on a `volume.Brickinfo`. If you have a particular reason to do it this way, let me know.